packages/tool-cache/__tests__/tool-cache.test.ts

```diff
@@ -268,96 +268,88 @@ describe('@actions/tool-cache', function() {
         await io.rmRF(tempDir)
```

Comment:
i had to recreate test.tar.gz with the option `--format pax` otherwise bsdtar doesnt preserve UTF file names correctly

Reply:
i added a README.txt to the archive for future reference